test/test-fs.c

```diff
@@ -2765,3 +2765,44 @@ TEST_IMPL(fs_read_write_null_arguments) {
 
   return 0;
 }
+
+TEST_IMPL(fs_file_pos_after_op_with_offset) {
+  int r;
+
+  /* Setup. */
+  unlink("test_file");
+  loop = uv_default_loop();
+
+  r = uv_fs_open(loop, &open_req1, "test_file", O_RDWR | O_CREAT,
+      S_IWUSR | S_IRUSR, NULL);
+  ASSERT(r >= 0);
+  ASSERT(open_req1.result >= 0);
```

Comment:
this assert is kinda redundant, since the result of a sync open is the same as the return value.